Formula/ssh-vault.rb

```diff
@@ -0,0 +1,32 @@
+class SshVault < Formula
+  desc "SSH encrypt/decrypt using ssh keys"
```

Comment:
The word "ssh" appears twice so I'm not sure I understand this description

Reply:
Can I just use `encrypt/decrypt using ssh keys` ?

Reply:
don't know why I put SSH at the beginning but I agree with you, I think was because a warking when running `brew audit --strict ssh-vault` but now seems to be ok.

Thanks for all your time checking this out 👍